server/delay_server/src/irodsDelayServer.cpp

```diff
@@ -674,6 +675,47 @@ namespace
 
         return {qstr, job};
     } // make_delay_queue_query_processor
+
+    auto is_current_server_leader() -> bool
```

Comment:
'current' server... perhaps... 'local' server instead?

`is_local_server_leader()`
or
with one more word...
`is_local_server_the_leader()`

Reply:
or `is_set_as_delay_server_leader`

Reply:
I find the implication introduced by removing "current" or "local" from the name a tad confusing. I like `is_local_server_defined_as_delay_server_leader` or something like that. Just my two cents

Reply:
That sounds fine.

Let's see what @FifthPotato thinks.

Reply:
yeps `is_local_server_defined_as_delay_server_leader` has all the wordses.

Reply:
Guess I'll add all the wordses!